apps/smart-contracts/core/test/PrePOMarket.test.ts

```diff
@@ -465,6 +465,98 @@ describe('=> prePOMarket', () => {
       }
     })
 
+    it('reverts if redemption = 0, redemption fee = 0%, and redeeming before market end', async () => {
```

Comment:
nit:
```suggestion
    it('reverts if redemption = 0, fee = 0%, and before market end', async () => {
```
'redemption'/'redeeming' is implied, same for tests below

Reply:
also change 'redemption = 0' to 'amounts = 0', similarly for below tets